src/mad_spark_alt/core/answer_extractor.py

```
@@ -6,16 +6,39 @@
 
 from .interfaces import GeneratedIdea
+from .json_utils import safe_json_parse
+from .llm_provider import LLMRequest, llm_manager
 
 logger = logging.getLogger(__name__)
 
 
+def _validate_prompt_content(prompt: str) -> str:
+    """Validate and sanitize prompt content for LLM safety."""
+    if not prompt or not prompt.strip():
+        raise ValueError("Prompt cannot be empty")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)